src/graphics/draw/DebugRenderer.cpp

```diff
@@ -660,8 +661,26 @@ void drawSystemScreen(OLEDDisplay *display, OLEDDisplayUiState *state, int16_t x
             snprintf(uptimeStr, sizeof(uptimeStr), " Uptime: %um", mins);
         textWidth = display->getStringWidth(uptimeStr);
         nameX = (SCREEN_WIDTH - textWidth) / 2;
-        display->drawString(nameX, getTextPositions(display)[line], uptimeStr);
+        display->drawString(nameX, getTextPositions(display)[line++], uptimeStr);
     }
+    // line++;
+    char api_state[50] = {0};
+    if (service->api_state == service->STATE_BLE)
+        snprintf(api_state, sizeof(api_state), "API State: BLE Connected");
```

Comment:
Can we truncate these to API: BLE Connected? I feel like it's pretty intuitive we're exposing the state.